Plotly.Blazor.Examples/Components/PieChart.razor

```diff
@@ -1,7 +1,8 @@
-﻿<PlotlyChart Id="TestId" Config="config" Layout="layout" Data="data" @ref="chart"/>
+﻿<PlotlyChart Id="TestId" Config="config" Layout="layout" Data="data" EmitEvents="events" @ref="chart" @onplotly_click_pie="OnClick" @onplotly_hover_pie="OnHover" />
```

Comment:
It seems a shame to have to specify events twice (in `EmitEvents` and the `onplotly...`), but I can't think of a way around it.